### PR TITLE
Update free-courses-ur.md

### PR DESCRIPTION
**Pull request by Ajay-singh1**
I have added a free Youtube course in Hindi/Urdu and it is completely free.

## What does this PR do?
This PR adds a free youtube course of javascript programming language and it is completely free. 

### Why is this valuable (or not)?
This is valuable especially for those who wants to study Javascript and wish to learn it for free in their native language i.e. Hindi/Urdu 

### How do we know it's really free?
It is available on youtube free of cost.

### For book lists, is it a book? For course lists, is it a course? etc.
This is a course and not a book. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
@eshellman Kindly Review This Pull Request And suggest if any change is needed. Thanks.

File: courses/free-courses-ur.md [hunk 1]
```diff
@@ -12,3 +13,8 @@
 ### HTML and CSS
 
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
```
Comment (eshellman) [inline courses/free-courses-ur.md:20]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
The playlist link https://www.youtube.com/playlist?list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf is in Hind list. If it's mostly urdu it should be here, not there.

Comment (Ajay-singh1) [reply courses/free-courses-ur.md:20]: Working on making the changes!

**Comment by Ajay-singh1** [comment]
Yeah Man I have changed it now!

**Comment by eshellman** [comment]
good, now in this PR, remove the playlist added here from the hindi list, thus moving it from there to here

**Comment by Ajay-singh1** [comment]
I have checked the hindi list and I don't think that these playlist are there we are good to go.

**Pull request closed without merging**